virtualizor.py

```diff
@@ -401,8 +401,11 @@ def register_nics(self, definition):
         i = 0
 
         for info in definition['nics']:
+            mac = info.get('mac', random_mac())
+            if mac == "none":
```

Comment:
This looks more like a hack to fix the format of `collect.py` output. I would prefer `collect.py` never return those `mac: none` entry.